docs/core/integration-quality-scale/rules/iqs046.md

```diff
@@ -0,0 +1,41 @@
+---
+title: "IQS046 - The documentation provides a list of known supported / unsupported devices"
+---
+
+IQS046 is as follows:
+
+The documentation provides a list of known supported / unsupported devices
+
+## Reasoning
+
+A lot of Home Assistant users buy devices based on if Home Assistant supports them.
+To make it easier for users to find out if a device is supported, the documentation should provide a list of known supported and unsupported devices.
+This would decrease the amount of bad experiences where the user finds out their device is not supported when they try to set it up.
+
+## Example implementation
+
+```markdown
+## Supported devices
+
+The following devices are known to be supported by the integration:
+- Device 1
+- Device 2
+- Every appliance that runs MyOS
+
+## Unsupported devices
+
+The following devices are not supported by the integration:
+- Device 3
+- Appliances built before 2010
+```
+
+## Additional resources
+
+
+## Exceptions
+
+This rule does not apply to integrations that do not connect to a device or service.
+This rule also does not apply to integrations that don't expose real life physical devices.
```

Comment:
So what I try to mean with this exception, is that integrations like `nyt_games` or `plex` or `tautulli` don't expose real life devices to Home Assistant. They just provide services, not physical devices.

But in the case of Withings, where we receive data on user level, not on device level, I do want to apply the rule because withings also produces cameras nowadays for some reason. Like, I do want to give a list of devices to tell the users, all these health devices are supported, and when new ones come out, I want to check if I implemented everything that that device provides so I can assure users that its usable.

Reply:
```suggestion
This rule also does not apply to integrations that don't integrate physical devices.
```

Reply:
```suggestion
This rule also does not apply to integrations that don't integrate physical devices.
```